src/main/java/org/jahia/modules/contenteditor/utils/ContentEditorUtils.java

```diff
@@ -181,16 +239,7 @@ private static boolean checkPermissionForType(ExtendedNodeType type, JCRNodeWrap
     }
 
     private static boolean isNodeType(List<String> nodeTypes, ExtendedNodeType type) {
-        if (nodeTypes != null) {
-            for (String nodeType : nodeTypes) {
-                if (type.isNodeType(nodeType)) {
-                    return true;
-                }
-            }
-            return false;
-        }
-
-        return true;
```

Comment:
You changed the logic of this function, now if List<String> nodeTypes is null it will crash NPE.
Actually the test ContentEditorUtilsTest.getNodetypesTest() is failing because of that

Reply:
Indeed, thank you, last commit fix the issue